src/parser/vast_parser.js

```diff
@@ -154,17 +155,17 @@ export class VASTParser extends EventEmitter {
    * @param {Object} options - The options to initialize a parsing sequence
    */
   initParsingStatus(options = {}) {
-    this.rootURL = '';
-    this.remainingAds = [];
-    this.parentURLs = [];
     this.errorURLTemplates = [];
-    this.rootErrorURLTemplates = [];
-    this.maxWrapperDepth = options.wrapperLimit || DEFAULT_MAX_WRAPPER_DEPTH;
     this.fetchingOptions = {
       timeout: options.timeout || DEFAULT_TIMEOUT,
-      withCredentials: options.withCredentials
+      withCredentials: options.withCredentials,
     };
-
+    this.maxWrapperDepth = options.wrapperLimit || DEFAULT_MAX_WRAPPER_DEPTH;
+    this.parentURLs = [];
+    this.parsingOptions = { allowMultipleAds: options.allowMultipleAds };
+    this.remainingAds = [];
+    this.rootErrorURLTemplates = [];
+    this.rootURL = '';
```

Comment:
Any reason why this re order is need? or was it generate by prettier 2.0?

Reply:
There's no real need, I just thought it would be more readable 🤷‍♂️